apps/src/javalab/javalab-settings.module.scss

```diff
@@ -0,0 +1,22 @@
+@use 'control-buttons.module';
+@import 'color.scss';
+
+.buttonWhite {
+  @include control-buttons.whiteControlButton;
+}
+
+.selected {
+  background-color: $lightest_gray;
+  border-color: $lightest_gray;
+  color: $dark_charcoal;
+}
+
+.main {
+  display: inline-block;
+  float: left;
+}
+
+.dropdown {
+  bottom: 40px;
+  margin-left: 5px;
+}
```

Comment:
Nit: Git complains about no newline. I'm wondering if we should have linting for this.